src/containers/Elements/NodeProviderPanels.js

```diff
@@ -74,30 +74,51 @@ class NodeProviderPanels extends PureComponent {
     }));
   }
 
-  isPanelOpen = index => this.state.panels[index];
+  showPanelAlwaysOpen = (panelIndex, show) => {
+    this.setState(previousState => ({
+      panelsAlwaysOpen: map(
+        previousState.panelsAlwaysOpen,
+        (item, index) => {
+          if (panelIndex === index) { return show; }
+          return item;
+        },
+      ),
+    }));
+  };
+
+  isPanelAlwaysOpen = index => this.state.panelsAlwaysOpen[index];
+
+  anyPanelAlwaysOpen = () => some(
+    map(this.state.panelsAlwaysOpen, (value, index) => this.isPanelAlwaysOpen(index)),
+    show => show === true,
+  );
+
+  isPanelOpen = index => this.state.panelsOpen[index];
 
   anyPanelOpen = () => some(
-    map(this.state.panels, (value, index) => this.isPanelOpen(index)),
+    map(this.state.panelsOpen, (value, index) => this.isPanelOpen(index)),
     open => open === true,
   );
 
   render() {
     const { panels, stage, prompt } = this.props;
 
     return (
-      <Panels minimise={!this.anyPanelOpen()}>
+      <Panels minimise={!this.anyPanelAlwaysOpen() && !this.anyPanelOpen()}>
```

Comment:
I think this needs a better name... This is an attempt to allow a side panel to show up, even if it is empty, so that you could drop a node from the main panel on it if you don't want it in the main panel anymore. There may be a case that the side panel(s) are empty, and should hide in general, but should show up as a droppable area if a valid node starts being dragged. That is what I'm trying to specify here -- whether this is an area that should be visible when nodes are being dragged, even if it is usually hiding.

Reply:
Makes sense, but what I mean is does it actually need to distinguish the two kinds of open like it does here?

```jsx
isPanelOpen(index) => { panelHasNodes(index) || panelIsDragging(index); }
isAnyPanelOpen() => panels.map(isPanelOpen);

...

<Panels minimise={!this.isAnyPanelOpen()}>
```

Reply:
Ah, yeah, that makes sense. :)